lib/actions/jlinkTargetActions.js

```diff
@@ -238,6 +240,8 @@ export function loadDeviceInfo(serialNumberInput, fullRead = false, eraseAndWrit
         const serialNumber = parseInt(serialNumberInput, 10);
         let info;
         try {
+            const { major, minor } = await getJLinkVersion(serialNumber);
+            logger.info(`Using JLink version ${major}.${minor}`);
```

Comment:
"JLink" in this context could mean a number of things so we could be more specific, e.g. "J-Link Software".